Pcap++/header/PcapFileDevice.h

```diff
@@ -127,6 +141,16 @@ namespace pcpp
 		*/
 		LinkLayerType getLinkLayerType() const { return m_PcapLinkLayerType; }
 
+		/**
+		 * @return The precision of the timestamps in the file.
+		*/
+		FileTimestampPrecision getTimestampPrecision() const { return m_Precision; }
```

Comment:
For better test coverage, maybe we can add tests for this method?

It'll actually be interesting to test it before opening the file and after opening it

Reply:
You only added tests for `PcapFileWriterDevice::getTimestampPrecision()` but not for `PcapFileReaderDevice::getTimestampPrecision()`...

Reply:
@seladb `libpcap` scales precision because of `pcap_open_offline_with_tstamp_precision` behaviour. So, I added a test and updated comments of reader function. Can you take a look again before merging for anything i missed?

https://www.tcpdump.org/manpages/pcap_open_offline.3pcap.html

> pcap_open_offline_with_tstamp_precision() takes an additional precision argument specifying the time stamp precision desired; if PCAP_TSTAMP_PRECISION_MICRO is specified, packet time stamps will be supplied in seconds and microseconds, and if PCAP_TSTAMP_PRECISION_NANO is specified, packet time stamps will be supplied in seconds and nanoseconds. If the time stamps in the file do not have the same precision as the requested precision, they will be scaled up or down as necessary before being supplied.